### ISSUE TITLE
Fix Create Netherite gear insulation and add it to more armor

### ISSUE THREAD

**Issue by mycroftjr** [LunaPixelStudios/SteamPunk#301]
### Describe the Bug

In config/cold_sweat/item_settings.toml, you can see that "create:netherite_backtack", "create:netherite_diving_helmet", and "create:netherite_diving_boots" were intended to basically grant heat immunity, and from photos on the discord, I can see that at one point, that did work. However, it no longer works for them atm. The `, "static", "{}"` sections should be removed from those entries so that the insulation works regardless of the NBT data on the items.

Additionally, this config (or other means) should be used to add insulation to other armor in the pack (optimally most of these would only grant complete immunity if you were wearing the full set (or at least a full set of some variety of vacuum-sealed armor pieces)m but that seems complicated to encode currently):

1. the full Ad Astra Space Suit set should grant complete cold immunity (it protects astronauts from the coldness of the vacuum of space IRL after all) and a little bit of heat insulation (Astronauts aren't being cooked in their suits IRL either from radiation)
2. The Space Suit upgrades (Netherite Space Suit and Jet Suit) should build on this by adding heat insulation, up to complete heat immunity with the Jet Suit (it's future tech!)
3. the full Aquamirae Three Bolt set should grant enough cold insulation to "protect you from freezing in the waters if the Ice Maze" as the Full Set Ability says
4. All Netherite armor and armor made from Netherite armor should boast at least the same high level of heat insulation, but idk about immunity - just because the armor itself isn't effected by lava doesn't mean lava can't seep through to your skin
5. Ignitium Armor especially should add very high heat insulation if not outright immunity - after all, the full set is intended to make you feel right at home while swimming in lava!
6. Other high-tier armor such as Astrumm could have default insulation granted as well?

You should probably also add some or all of the above items to the "Insulation Blacklist", but especially _**all**_ Create Backtanks, since they lose any insulation you add to them when you place them in-world for refilling anyways.

### Reproduction Steps

1. Craft Netherite Backtank or Diving gear
3. Observe lack of insulation

### Operating System

Windows 10

### Minecraft Version

1.19.2

### Modloader

Forge

### Modpack Version

v42

### Example config
```
"Insulating Armor" = [["minecraft:leather_helmet", 4, 4], ["minecraft:leather_chestplate", 6, 6], ["minecraft:leather_leggings", 5, 5], ["minecraft:leather_boots", 4, 4], ["cold_sweat:hoglin_headpiece", 0, 8], ["cold_sweat:hoglin_tunic", 0, 12], ["cold_sweat:hoglin_trousers", 0, 10], ["cold_sweat:hoglin_hooves", 0, 8], ["cold_sweat:fur_cap", 8, 0], ["cold_sweat:fur_parka", 12, 0], ["cold_sweat:fur_pants", 10, 0], ["cold_sweat:fur_boots", 8, 0], ["environmental:yak_pants", 7.5, -5], ["create:netherite_backtank", 0, 1998], ["create:netherite_diving_helmet", 0, 1998], ["create:netherite_diving_boots", 0, 1998], ["ad_astra:space_helmet", 42, 0], ["ad_astra:space_suit", 63, 0], ["ad_astra:space_pants", 52.5, 0], ["ad_astra:space_boots", 42, 0], ["ad_astra:netherite_space_helmet", 42, 42], ["ad_astra:netherite_space_suit", 63, 63], ["ad_astra:netherite_space_pants", 52.5, 52.5], ["ad_astra:netherite_space_boots", 42, 42], ["ad_astra:jet_suit_helmet", 420, 420], ["ad_astra:jet_suit", 630, 630], ["ad_astra:jet_suit_pants", 525, 525], ["ad_astra:jet_suit_boots", 420, 420], ["cataclysm:ignitium_helmet", 0, 420], ["cataclysm:ignitium_chestplate", 0, 630], ["cataclysm:ignitium_elytra_chestplate", 0, 630], ["cataclysm:ignitium_leggings", 0, 525], ["cataclysm:ignitium_boots", 0, 420]]
...
"Insulation Blacklist" = ["create:netherite_backtank", "create:netherite_diving_helmet", "create:netherite_diving_boots", "ad_astra:jet_suit_helmet", "ad_astra:jet_suit", "ad_astra:jet_suit_pants", "ad_astra:jet_suit_boots"]
```

**Comment by SHXRKIEgg** [comment]
Will fix in next update thank you

**Comment by MikulDev** [comment]
It should be noted that the stuff about the netherite diving gear is a bug with Cold Sweat. It will be fixed in the next version (2.3-b04h)
